oneview_redfish_toolkit/multiple_oneview.py

```diff
@@ -137,6 +158,8 @@ def search_resource_multiple_ov(resource, function, resource_id,
             resource: resource type (server_hardware)
             function: resource function name (get_all)
             resource_id: set only if it should look for a specific resource ID
+            ov_ips: List of Oneview IPs to search the resource. If None is
+                passed, the the list of IPs will be all IPs configured
```

Comment:
ov_ips: List of Oneview IPs to search for the resource. If None is passed, it will search in all IPs based on the toolkit configuration.